### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by paulpestov**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,5 @@ work/
 workflows/nf-results/*
 workflows/results
 workflows/ocrd-workflows/*.nf
-models
\ No newline at end of file
+models
+.idea
```
Comment (mweidling) [inline .gitignore:11]: ?

File: README.md [hunk 2]
```diff
@@ -17,18 +17,18 @@ To speed up QuiVer Benchmarks you can mount already downloaded text recognition
...
-Otherwise the tool will download all `ocrd-tesserocr-recognize` models as well as `ocrd-calamari-recognize qurator-gt4histocr-1.0` on each run.
+Otherwise, the tool will download all `ocrd-tesserocr-recognize` models as well as `ocrd-calamari-recognize qurator-gt4histocr-1.0` on each run.
 
 ## Usage
 
 - clone this repository
-- [customize](#custom-workflows-and-data) QuiVer Benchmarks according to your needs
-- run `docker compose build && docker compose up`
+- (optional) [customize](#custom-workflows-and-data) QuiVer Benchmarks according to your needs
+- run `docker-compose up --build`
```
Comment (mweidling) [inline README.md:26]: I should have specified in the prerequisites that I use the Compose plugin. No need for the dash then.

File: README.md [hunk 3]
```diff
@@ -41,25 +41,25 @@ The relevant benchmarks gathed by QuiVer Benchmarks are defined in [OCR-D's Qual
...
 
 Add new OCR-D workflows to the directory `workflows/ocrd_worflows` according to the following conventions:
 
-- OCR workflows have to end with `_ocr.txt`, evaluation workflows with `_eval.txt`. The files will be converted by OtoN to Nextflow files after the container has started.
+- OCR workflows have to end with `_ocr.txt`, evaluation workflows with `_eval.txt`. The files will be converted by [OtoN](https://github.com/MehmedGIT/OtoN_Converter) to Nextflow files after the container has started.
 - workflows have to be TXT files
 - all workflows have to use [`ocrd process`](https://ocr-d.de/en/user_guide#ocrd-process)
 
-You can then either rebuild the Docker image via `docker compose build` or mount the directory to the container via
+You can then either rebuild the Docker image via `docker-compose build` or mount the directory to the container via
```
Comment (mweidling) [inline README.md:56]: See above.

File: README.md [hunk 4]
```diff
@@ -41,25 +41,25 @@ The relevant benchmarks gathed by QuiVer Benchmarks are defined in [OCR-D's Qual
...
 
 ```yml
 - ./workflows/ocrd_workflows:/app/workflows/ocrd_workflows
 ```
 
-in the `volumes` section and spin up a new run with `docker compose up`.
+in the `volumes` section and spin up a new run with `docker-compose up`.
```
Comment (mweidling) [inline README.md:62]: See above.

File: README.md (hunk 2 above)
Comment (paulpestov) [reply README.md:26]: Hm I thought the `docker-compose` is more common for not experienced users (like me). Having only one well known command would eliminate possible hurdles.

File: .gitignore (hunk 1 above)
Comment (paulpestov) [reply .gitignore:11]: For JetBrains users ;)

Comment (kba) [reply .gitignore:11]: IntelliJ configuration

File: README.md (hunk 2 above)
Comment (kba) [reply README.md:26]: my 2ct :+1: for `docker-compose`

Comment (MehmedGIT) [reply README.md:26]: `docker-compose` is V1 and `docker compose` is V2. Check [here](https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose). The V1 will/may eventually get dropped at some point, so I think it's better to stick to V2.

Comment (paulpestov) [reply README.md:26]: Ok wow, [here](https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command) official statement linked in some answer there. Maybe we should at least add at point how to install `compose`.

Comment (MehmedGIT) [reply README.md:26]: Honestly, I did not expect it to be that soon :)

Comment (mweidling) [reply README.md:26]: > Ok wow, [here](https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command) is an official statement linked in some answer there. Maybe we should at least add a point how to install `compose`.

Sure, I honestly just forgot to put it in the requirements. :)

File: .gitignore (hunk 1 above)
Comment (mweidling) [reply .gitignore:11]: Alrighty!

File: README.md (hunk 2 above)
Comment (mweidling) [reply README.md:27]: Added in [572e248](https://github.com/OCR-D/quiver-benchmarks/pull/1/commits/572e248a3dc4c89761414be5edeca76b8d0f1ed7)

File: README.md (hunk 3 above)
Comment (mweidling) [reply README.md]: Changed in [27d7e5a](https://github.com/OCR-D/quiver-benchmarks/pull/1/commits/27d7e5ae8cdaf5a8dc817cbfd7be37dfc6d497f5) (for all instances).

**Comment by mweidling** [comment]
Merci!

**Pull request merged**